storage_test.go

```diff
@@ -105,3 +113,42 @@ func TestUnlock(t *testing.T) {
 	_, err = s.bucket.Object("a.lock").Attrs(context.Background())
 	assert.ErrorAs(t, err, &storage.ErrObjectNotExist)
 }
+
+func TestEncryption(t *testing.T) {
+	s := setupTestStorage(t, []fakestorage.Object{
+		{
+			ObjectAttrs: fakestorage.ObjectAttrs{
+				BucketName: testBucket,
+				Name:       "some/object/",
+			},
+		},
+	})
+	kh, err := keyset.NewHandle(aead.AES256GCMKeyTemplate())
+	assert.NoError(t, err)
+	kp, err := aead.New(kh)
+	assert.NoError(t, err)
+
+	s.aead = kp
```

Comment:
this assignment seems to be ineffectual, am I missing something?

Reply:
Good point!